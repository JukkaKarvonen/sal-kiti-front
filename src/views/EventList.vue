<template>
  <div class="home">
    <EventList />
  </div>
</template>

<script>
/**
 * Event list view
 */
import EventList from "@/components/EventList.vue";

export default {
  name: "EventListView",
  components: {
    EventList
  },
  created() {
    document.title = this.$t("title.events") + " | " + this.$t("title.prefix");
  }
};
</script>
