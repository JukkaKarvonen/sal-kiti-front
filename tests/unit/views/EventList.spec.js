import { shallowMount, createLocalVue } from "@vue/test-utils";
import EventList from "@/views/EventList.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("EventList.vue", () => {
  const $route = {
    params: { id: 1 }
  };
  it("renders content", () => {
    const wrapper = shallowMount(EventList, {
      localVue,
      mocks: {
        $route
      }
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
