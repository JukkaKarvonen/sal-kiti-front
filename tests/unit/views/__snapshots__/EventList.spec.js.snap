// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventList.vue renders content 1`] = `
<div
  class="home"
>
  <eventlist-stub />
</div>
`;
